docs/sphinx/formats/options.rst

```diff
@@ -0,0 +1,40 @@
+Additional Reader options
+=========================
+
+Some Readers have additional options which can be used to inform how
```

Comment:
Having checked, no, only when referring to a specific FormatReader so changed there & in page title